libbeat/processors/actions/docs/replace.asciidoc

```diff
@@ -35,11 +35,11 @@ processors:
 
 The `replace` processor has following configuration settings:
 
-`ignore_missing`:: (Optional) If set to true, no error is logged in case a specifiedfield 
+`ignore_missing`:: (Optional) If set to `true`, no error is logged in case a specified field 
 is missing. Default is `false`.
 
-`fail_on_error`:: (Optional) If set to true, in case of an error the replacement of
-field values is stopped and the original event is returned. If set to false, replacement
+`fail_on_error`:: (Optional) If set to `true`, in case of an error the replacement of
+field values is stopped and the original event is returned. If set to `false`, replacement
 continues even if an error occurs during replacement. Default is `true`.
```

Comment:
You made a good update to a similar config variable in another PR. I think it was something like this:

```suggestion
`fail_on_error`:: (Optional) If set to `true` and there's an error, the replacement of
field values is stopped and the original event is returned. If set to `false`, replacement
continues even if an error occurs during replacement. Default is `true`.
```